internal/server/models/pagination.go

```diff
@@ -0,0 +1,84 @@
+package models
+
+import (
+	"math"
+	"strings"
+
+	"github.com/infrahq/infra/api"
+)
+
+// Internal Pagination Data
+type Pagination struct {
+	Page  int
+	Limit int
+	Sort  string
+	Total int
+	Pages int
+	Next  int
+	Prev  int
```

Comment:
What are the use cases for these fields?

Reply:
If there is no next or previous page then the field would be empty, other than that it's just a ±1. I guess they're not very necessary